### PR TITLE
Update README.md file with accesibility

### PR DESCRIPTION
**Pull request by rmrmrmr**
## In this capstone we:

Evaluated the website against accessibility best practices. Specifically, we checked the following points:
- Page titles - no issues found
- Image text alternative - changes indicated in the comments below
- Text Headings - no issues found
- Color contrast - no issues found
- Resize - no issues found
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

This makes the webpage more accessible for anyone regardless on how they choose to interact with it! 🚀🙌

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h2 class="primaryText" id="primaryJob">
   <section id="recentWorks">
     <div id="recentWorksHeader">
       <p id="rWrksTitle">My Recent Works</p>
-      <img id="indicatorRW" src="Images/Indicator.png" alt="Indicator">
+      <img id="indicatorRW" src="Images/Indicator.png" alt="Page divider">
```
Comment (rmrmrmr) [inline index.html:75]: Changed the alt text from "Indicator" to "Page divider"

File: index.html [hunk 2]
```diff
@@ -285,7 +285,7 @@ <h3 id="abtMeText" class="abtMeText">
     </div>
     <div id="languagesCard" class="abtMeCard">
       <div class="langIconPH">
-        <img src="Images/icon-languages.svg" alt="Languages">
+        <img src="Images/icon-languages.svg" alt="Languages icon">
```
Comment (rmrmrmr) [inline index.html:288]: Changed the alt text from "Languages" to "Languages icon"

File: index.html [hunk 3]
```diff
@@ -313,7 +313,7 @@ <h3 id="abtMeText" class="abtMeText">
     </div>
     <div id="frameworksCard" class="abtMeCard">
       <div class="langIconPH">
-        <img src="Images/icon-frameworks.svg" alt="Languages">
+        <img src="Images/icon-frameworks.svg" alt="Frameworks icon">
```
Comment (rmrmrmr) [inline index.html:316]: Changed the alt text from "Languages" to "Frameworks icon"

File: index.html [hunk 4]
```diff
@@ -346,7 +346,7 @@ <h3 id="abtMeText" class="abtMeText">
     </div>
     <div id="skillsCard" class="abtMeCard">
       <div class="langIconPH">
-        <img src="Images/icon-skills.svg" alt="Languages">
+        <img src="Images/icon-skills.svg" alt="Skills icon">
```
Comment (rmrmrmr) [inline index.html:349]: Changed the alt text from "Languages" to "Skills icon"

**Review by frank1738 — APPROVED** [review]
Hi @rmrmrmr , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**